code/game/objects/items/sticker.dm

```diff
@@ -89,5 +89,7 @@
 	set src in view(1)
 
 	var/obj/item/sticker/S = locate() in src
+	if(!isliving(usr))
+		return
```

Comment:
```suggestion

	if(!isliving(usr))
		return
	var/obj/item/sticker/S = locate() in src
```
We don't need to find the sticker if the mob isn't living anyway. Only if the mob is living should we care.